### ISSUE TITLE
Using wildcards ends up with exception

### ISSUE THREAD

**Issue by mregen** [novotnyllc/NuGetKeyVaultSignTool#78]
```
D:\codesign>NuGetKeyVaultSignTool sign **/OPCFoundation.*.nupkg --file-digest sha256 --timestamp-rfc3161 http://timestamp.digicert.com --timestamp-digest sha256 --azure-key-vault-url <> --azure-key-vault-client-id <> --azure-key-vault-tenant-id <> --azure-key-vault-client-secret "" --azure-key-vault-certificate xyz
info: NuGetKeyVaultSignTool.Program[0]
      SignAsync [D:\codesign\1.4.363-20200904.2-netcore3-preview\opcfoundation.netstandard.opc.ua.symbols.1.4.363-20200904.2-netcore3-preview.nupkg]: Begin Signing opcfoundation.netstandard.opc.ua.symbols.1.4.363-20200904.2-netcore3-preview.nupkg
warn: NuGetKeyVaultSignTool.Program[0]
      NuGet [D:\codesign\1.4.363-20200904.2-netcore3-preview\opcfoundation.netstandard.opc.ua.symbols.1.4.363-20200904.2-netcore3-preview.nupkg]: NU3018: A certificate chain processed, but terminated in a root certificate which is not trusted by the trust provider.
fail: NuGetKeyVaultSignTool.Program[0]
      The filename, directory name, or volume label syntax is incorrect. : 'D:\codesign\**\OPCFoundation.*.nupkg'
System.IO.IOException: The filename, directory name, or volume label syntax is incorrect. : 'D:\codesign\**\OPCFoundation.*.nupkg'
   at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
   at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at System.IO.File.Open(String path, FileMode mode, FileAccess access)
D:\codesign>NugetKeyVaultSignTool --version
```

fix --> #77

**Comment by mregen** [comment]
Hi @clairernovotny, would be great to get a fixed version released. Thanks, Martin

**Comment by mregen** [comment]
Add reference to https://github.com/OPCFoundation/UA-.NETStandard/pull/1122

**Comment by clairernovotny** [comment]
Fixed in 3.0.45, thanks!
